### ISSUE TITLE
Drilljig missing in 0.6

### ISSUE THREAD

**Issue by steampunklolcat** [newHeiko/wiFred#47]
The drilljig is missing from the 0.6 releases of the gerber-files.

**Comment by newHeiko** [comment]
True, we started creating laser engraved housings somewhere around 0.5. But I'll have a look into creating the drilljig again.

Heiko
